providers/oidc_test.go

```diff
@@ -373,3 +403,22 @@ func TestOIDCProvider_findVerifiedIdToken(t *testing.T) {
 	assert.Equal(t, nil, err)
 	assert.Equal(t, true, verifiedIDToken == nil)
 }
+
+func TestAppendStringGroup(t *testing.T) {
```

Comment:
Once you rename/refactor this method per my note above, for private testers - we usually do `Test_methodName` to handle the lower case private methods.

Lets also merge these two test methods into 1 method with a test table.

Reply:
Eventually one of the maintainers will likely circle back to this file and refactor the whole thing in Ginkgo since that is where we are moving our test suites.